test/wpt/README.md

```diff
@@ -91,7 +91,11 @@ add this to `test/wpt/status/url.json`:
 
 ```json
   "url-searchparams.any.js": {
-    "fail": "explain why the test fails, ideally with links"
+    "fail": {
+      "expected": [
+        "test name"
```

Comment:
I think PR technically changes what the test spec means - when `fail` is a string, the value is a generic description of why we expect the test to fail (which is not necessarily related to the WPT test description), but when it's an object shaped like `{"expected": [...]}`, the elements of the array would be used to match against the actual test descriptions in the WPT. Can you keep the documentation for the original schema, add some descriptions about the new one in the README?

Reply:
The original schema meant that any truthy fail value would make any failure, even a new one, an expected failure.

In my opinion this doesn't work for a test runner.

That's why I changed the fail schema and made what were effectively skips (wrt overall process result) an actual explicit skip.